xml/Microsoft.Extensions.Logging/LoggerFactory.xml

```diff
@@ -29,7 +29,7 @@
     </Interface>
   </Interfaces>
   <Docs>
-    <summary>Produces instances of <see cref="T:Microsoft.Extensions.Logging.ILogger" /> classes based on the given providers.</summary>
+    <summary>Produces instances of <see cref="T:Microsoft.Extensions.Logging.ILogger" /> classes based on the specified providers.</summary>
```

Comment:
@gewarren @carlossanlop - for some reason I thought the triple-slash doc comments in dotnet/runtime were now the "master". Shouldn't we be updating them instead?

https://github.com/dotnet/runtime/blob/a8ac67ae5373dce4d337012d2e8c875d32ffa638/src/libraries/Microsoft.Extensions.Logging/src/LoggerFactory.cs#L12

Reply:
They are not yet. We have only moved over 3 projects and we stopped because the porting tool has some issues. However we will try and make it the main story for .NET 7

Reply:
Is the plan to port these changes to the triple-slash doc comments in the code?

Reply:
We should with the doc porting tool once that is ready.